testdata/database_create.json

```diff
@@ -41,7 +41,7 @@
     }
   ],
   "parent": {
-    "type": "page_id",
```

Comment:
same here. This test data is still valid and have not to be changed

Reply:
Done